### DISCUSSION TITLE
Trigger a job each 5 seconds starting at a full minute

### DISCUSSION THREAD

**Discussion by GerhardKreuzer** [quartznet/quartznet#766] [General] [answered]
Hi,
I want to Trigger a Job every 5 seconds, starting at hh.mm.(ss mod 5).
All I tried Ends up in starting the Job each 5 seconds after starting the Scheduler, so it fires at 7:53:07, ...12, ..17 and so on.
Is there a Chance to define a Trigger to start at the next second mod 5?
So when I start the Scheduler at 7:53:07 the Trigger schould fire first at 7:53:10 and than every 5 seconds.

Thanks for helping.

With best regards

Gerhard

PS: I use the latest Version!!

**Comment by lahma** [comment, marked as answer]
You need to set the start time accordingly, 5 second interval is calculated from there. For example:

```
var trigger = TriggerBuilder.Create()
    .StartAt(DateBuilder.EvenMinuteDateAfterNow())
    .WithSimpleSchedule(x => x.WithIntervalInSeconds(5).RepeatForever())
    .Build();
```

StartAt accepts DateTimeOffset that you can calculate yourself.
